<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QDialog" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="sizeLabel">
      <property name="text">
       <string>Give the board size:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="seedSpinBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="goalLabel">
      <property name="text">
       <string>Give a goal value:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="sizeSpinBox"/>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="mSpinBox"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="seedLabel">
      <property name="text">
       <string>Give a seed value:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="goalSpinBox"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="mLabel">
      <property name="text">
       <string>Give the value of m:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="finishPushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="errorTextBrowser">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Block a tile</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="blockRadioButton">
       <property name="text">
        <string>Yes</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="nblockRadioButton">
       <property name="text">
        <string>No</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>271</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QLabel" name="yblockLabel">
      <property name="text">
       <string>y</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="xblockLabel">
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>x</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="xblockSpinBox"/>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="yblockSpinBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
